pkg/connProfile/get.go

```diff
@@ -0,0 +1,123 @@
+/*
+Copyright 2023 The Bestchains Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package connProfile
+
+import (
+	"context"
+	"encoding/base64"
+	"encoding/json"
+	"fmt"
+	"github.com/bestchains/bc-cli/pkg/common"
+	"github.com/spf13/cobra"
+	"github.com/spf13/viper"
+	"gopkg.in/yaml.v2"
+	v1 "k8s.io/apimachinery/pkg/apis/meta/v1"
+	"k8s.io/apimachinery/pkg/apis/meta/v1/unstructured"
+	"k8s.io/apimachinery/pkg/runtime/schema"
+	"k8s.io/client-go/dynamic"
+	"k8s.io/client-go/tools/clientcmd"
+)
+
+func getNestedString(obj map[string]interface{}, fields ...string) string {
+	val, found, err := unstructured.NestedString(obj, fields...)
+	if !found || err != nil {
```

Comment:
I think here it could just be written as:

```go
val, _, _ := unstructured.NestedString(obj, fields...)
return val
```
Looking at the `NestedString` function implementation, `val` is the empty string when there is an error or the field does not exist.